epl-nft/lib/go/test/epl_test.go

```diff
@@ -375,9 +375,11 @@ func testCreateEdition(
 		assert.Equal(t, setID, edition.SetID)
 		assert.Equal(t, playID, edition.PlayID)
 		assert.Equal(t, tier, edition.Tier)
-		//if maxMintSize != nil {
-		//	assert.Equal(t, &maxMintSize, &edition.MaxMintSize)
-		//}
+		if maxMintSize != nil {
+			assert.Equal(t, &maxMintSize, &edition.MaxMintSize)
+		} else {
+			assert.Equal(t, uint64(0), *edition.MaxMintSize)
+		}
```

Comment:
@judezhu added this to ensure maxMintSize logic in the if block you pointed out is working as designed. MaxMintSize is returning as I would expect it to for both a legit val and nil.